packages/core/src/bin/commands/createViews.ts

```diff
@@ -0,0 +1,158 @@
+import { createBuild } from "@/build/index.js";
+import { createDatabase, getPonderMeta } from "@/database/index.js";
+import { createLogger } from "@/internal/logger.js";
+import { MetricsService } from "@/internal/metrics.js";
+import { buildOptions } from "@/internal/options.js";
+import { createShutdown } from "@/internal/shutdown.js";
+import { createTelemetry } from "@/internal/telemetry.js";
+import { eq, sql } from "drizzle-orm";
+import type { CliOptions } from "../ponder.js";
+import { createExit } from "../utils/exit.js";
+
+const emptySchemaBuild = {
+  schema: {},
+  statements: {
+    tables: { sql: [], json: [] },
+    enums: { sql: [], json: [] },
+    indexes: { sql: [], json: [] },
+  },
+};
+
+export async function createViews({
+  cliOptions,
+}: {
+  cliOptions: CliOptions & {
+    schema?: string | undefined;
+    viewsSchema?: string | undefined;
+  };
+}) {
+  const options = buildOptions({ cliOptions });
+
+  const logger = createLogger({
+    level: "warn",
+    mode: options.logFormat,
+  });
+
+  const metrics = new MetricsService();
+  const shutdown = createShutdown();
+  const telemetry = createTelemetry({ options, logger, shutdown });
+  const common = { options, logger, metrics, telemetry, shutdown };
+
+  const build = await createBuild({ common, cliOptions });
+
+  const exit = createExit({ common });
+
+  if (cliOptions.schema === undefined) {
```

Comment:
We should also probably validate that these aren't empty strings?

Reply:
The CLI tool validates this